nautobot/core/templatetags/helpers.py

```diff
@@ -756,7 +756,7 @@ def saved_view_modal(
             if current_saved_view_pk:
                 current_saved_view_pk = current_saved_view_pk[0]
                 try:
-                    current_saved_view = SavedView.objects.restrict(request.user, "view").get(pk=current_saved_view_pk)
+                    current_saved_view = SavedView.objects.get(pk=current_saved_view_pk)
```

Comment:
Mind adding a comment here to the effect that we're intentionally *not* calling `restrict()` here and why?